package-lock.json

```diff
@@ -10862,8 +10862,8 @@
       }
     },
     "wet-boew": {
-      "version": "github:wet-boew/wet-boew#ca91e24693eff3f4375969dbbf9bbabc61652079",
-      "from": "github:wet-boew/wet-boew#v4.0.50.1",
+      "version": "github:wet-boew/wet-boew#",
```

Comment:
Let's rebase,
The commit ID should be: 43c6f33767a2cdb4bb286cee62b2dc0a181d0f10